### PR TITLE
Add links for cards

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**
Added Card Links to the side-Bar, when clicked it scrolls down to the specific card.
Also Added Save and Cancel buttons to the cards.

### CODE REVIEW

File: app/javascript/Clients/clientIdPage/ClientIdPage.js [hunk 1]
```diff
@@ -11,34 +11,96 @@ import {
...
 
-const navLinks = (
-  <NavLinks>
-    <NavLink text="Client Information" href="#ClientInformation" />
-    <NavLink text="Safety Alert Information" href="#SafetyAlertInformation" />
-    <NavLink text="Other Client Information" href="#Other CLient Information" />
-    <NavLink text="Race & Ethinicity" href="#Race & Ethinicity" />
-  </NavLinks>
-);
+export default class ClientIdPage extends React.PureComponent {
```
Comment (RamuKammagani-osi) [inline app/javascript/Clients/clientIdPage/ClientIdPage.js]: Are we using React.pureComponent?? Can you please check this once again?

Comment (Sushmareddy-OSI) [reply app/javascript/Clients/clientIdPage/ClientIdPage.js]: No, I was trying to use. we are still using React.component

Comment (RamuKammagani-osi) [reply app/javascript/Clients/clientIdPage/ClientIdPage.js]: Okay! if not! Can you refactor the code.

**Comment by mlipardo** [comment]
The links on the side bar are not the same order with the cards.

**Comment by NagaPradeepGajula-osi** [comment]
CodeClimate rules were refactored to address this issue, but it seems its not working. The team is ok with this so merging the PR.

**Pull request merged**